src/Toml/Bi/Combinators.hs

```diff
@@ -254,5 +252,5 @@ table codec key = Codec input output
     handleTableName e                         = throwError e
 
 -- | Used for @newtype@ wrappers.
```

Comment:
While we're on it, you also can write nice documentation example on how to use this function :slightly_smiling_face: